projects/crud/src/lib/components/listing/listing.component.html

```diff
@@ -164,4 +164,6 @@ <h2 mat-dialog-title style="padding:10px;" class="separated" *ngIf="viewConfig.d
         *ngIf="viewConfig.pagination.enabled" [length]="resultsCount" [pageIndex]="pages-1" [pageSize]="20"
         (page)="onChange($event)">
     </mat-paginator>
-</ng-container>
\ No newline at end of file
+    <ng-container *ngIf="openGrid">
+            <div #customGrid></div>
```

Comment:
@lamiselzobeidy let's change all the flags names to be generic, not related to grids at all (openCustomComponent, #customComponent)